platform/spec/controllers/api/v1/connections_controller_spec.rb

```
@@ -82,4 +82,77 @@
+  describe "#start_sync" do
+    let(:connection) do
+      create(:connection,
+                              workspace: workspace,
+                              source: source_connector,
+                              destination: destination_connector,
+                              status: :created)
+    end
+
+    context "when successful" do
+      before do
+        allow_any_instance_of(Connections::StartDataSyncService)
```

rubocop (1.68.0)

[convention] Avoid stubbing using `allow_any_instance_of`.

(RSpec/AnyInstance)